docs/UserGuide.md

```diff
@@ -663,3 +686,14 @@ If you want to end the application, simply type `exit`, or click on the `Exit` b
 ## Troubleshooting
 
 ### How to check your Java version
+
+#### For **Windows** Users
+{Kenny and Sean pls}
+
+#### For **Mac** and **Linux** Users
+**Step 1** Open up Terminal.
+
+**Step 2** Run the command `java -version`
+
+**Step 3** Check the version number from the output.
```

Comment:
Not sure if we want to add in `which java` as well if the user is unable to find the executable due to file path issues